src/test.core.c

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+#include <stdio.h>
+
+// Include all headers.
+#include <openassetio/c/InfoDictionary.h>
+#include <openassetio/c/StringView.h>
+#include <openassetio/c/errors.h>
+#include <openassetio/c/hostApi/Manager.h>
+#include <openassetio/c/managerApi/CManagerInterface.h>
+#include <openassetio/c/managerApi/HostSession.h>
+#include <openassetio/c/managerApi/ManagerInterface.h>
+#include <openassetio/c/namespace.h>
```

Comment:
[plan] Can we update the documentation in the main repo to indicate that we need to update this list when we work on the c module?

Reply:
Added https://github.com/OpenAssetIO/OpenAssetIO/issues/713